backend/app/feedback/schemas.py

```diff
@@ -14,11 +14,22 @@ class Feedback(BaseModel):
     user_name: str
     chat_id: UUID
     feedback_text: str
+    like: bool
     created_datetime: datetime = Field(default_factory=datetime.now)
 
     model_config = ConfigDict(from_attributes=True)
 
 
+class FeedbackRequest(BaseModel):
```

Comment:
Suggest `Feedback` inherits from "FeedbackRequest" since it seems to be superset of fields